app/Repositories/Project/ProjectRepository.php

```diff
@@ -1074,4 +1090,51 @@ private function getActivityGrade($projectId, $activityParam)
         return null;
 
     }
+
+    public function getProjects($suborganization, $data) {
+
+        $authenticated_user = auth()->user();
+        $query = $authenticated_user->projects();
+
+//        if (isset($data['skipPagination']) && $data['skipPagination'] === 'true') {
+        if (!isset($data['size'])) {
+            return $query->where('organization_id', $suborganization->id)
+                ->whereNull('team_id')
+                ->orderBy('order', 'ASC')->get();
+        }
+
+        $perPage = isset($data['size']) ? $data['size'] : config('constants.default-pagination-per-page');
+
+        if (isset($data['order_by_column'])) {
+            $orderByType = isset($data['order_by_type']) ? $data['order_by_type'] : 'ASC';
+            $query = $query->orderBy($data['order_by_column'], $orderByType);
+        } else {
+            $query = $query->orderBy('order', 'ASC');
+        }
+
+        return $query->where('organization_id', $suborganization->id)->whereNull('team_id')->paginate($perPage)->withQueryString();
+    }
+
+    public function getFavoriteProjects($suborganization, $data) {
+
+        $authenticated_user = auth()->user();
+        $query = $authenticated_user->favoriteProjects();
+
+//        if (isset($data['skipPagination']) && $data['skipPagination'] === 'true') {
+        if (!isset($data['size'])) {
+            return $query->wherePivot('organization_id', $suborganization->id)
```

Comment:
The where conditions below are common so can be moved out of if.

->wherePivot('organization_id', $suborganization->id)